### ISSUE TITLE
It's difficult to click the points on the left image.

### ISSUE THREAD

**Issue by willieChen58** [florianblume/6d-pat#165]
I miss a problem just like that : when i'm clicking the points on the left image, it's so hard to achieve a success. I must try many times to get a point on the left image, but it's easy to click the points on the right model. Would you miss the problem like that? I wonder if it's only my problem, looking forward to your reply.

**Comment by willieChen58** [comment]
![image](https://user-images.githubusercontent.com/128902725/232656346-34bd24be-7797-496c-8295-aa936a7b372e.png)

**Comment by florianblume** [comment]
Does the status text of the program in the bottom left corner change when you click the image? I.e. is does it say image clicked? If so, then maybe the click just doesn't get shown and I'll know better where the problem lies.

**Comment by florianblume** [comment]
Did you check that the mouse button you use is the one set in the settings? Maybe there's a bug there that after some time you're able to click the image with the wrong button even if that's not supposed to happen.

**Comment by willieChen58** [comment]
Yes, I've checked my setttings, and I did use the left mouse buton to click the image as what I set.

**Comment by willieChen58** [comment]
![image](https://user-images.githubusercontent.com/128902725/233013943-de704780-cad6-4056-b8ac-aab40af32684.png)

**Comment by florianblume** [comment]
Hm, it's difficult to say from the distance what it could be then. Are you able to clone the project and build it locally using Qt? Then I could tell you where to set breakpoints so that we can go through the code remotely so to say.

**Comment by willieChen58** [comment]
Well, I don‘t konw how to clone the project and build it locally using Qt. I am thinking about whether it's because i don't install the requirements that you mention in the readme, but i do run the Appimage and create the pose data without the requirements.

**Comment by florianblume** [comment]
Doesn't sound like the requirements, in that case the program wouldn't start. I don't think I can help you properly like this unless you're able to clone the project and build it. It's not that difficult, maybe you find a guide how to clone projects from github.

**Comment by willieChen58** [comment]
Thank you so much ! I will learn to how to clone the project and build it  following your instructions as far as i can.

**Comment by florianblume** [comment]
Alright, let me know when you've succeeded!

**Comment by willieChen58** [comment]
OK !

**Comment by florianblume** [comment]
Have you succeeded in getting everything setup?
